README.md

```diff
@@ -45,23 +45,29 @@ See [the tests](test/run-spec.js) for more examples!
 
 ### Batch run
 
-`request.json`:
-```json
-[
-  {
-    "initialBoard": "GBB...",
-    "originalCode": "procedure A() { ...",
-    "code": "program { ..."
-  },
-  {
-    "initialBoard": "GBB...",
-    "code": "program { ..."
-  },
-]
+`batch.json`:
+
+```js
+{
+  "code": "procedure Meter(color) { Poner(color) }", // Student's code
+  "extraCode": "function a() { return (2) }", // [OPTIONAL] Teacher's code
+  "examples": [
+    {
+      "initialBoard": "GBB/1.0\nsize 4 4\nhead 0 0", // Initial board
+      "extraBoard": "GBB/1.0\nsize 4 4\nhead 0 0", // [OPTIONAL] Expected board
+      "generatedCode": "procedure Meter(color) { Poner(color) } program { Meter(Azul) }" // [OPTIONAL] Code to be executed, overrides `code`
```

Comment:
Just curious: why are you naming it `generatedCode` and not just `code`?

Reply:
It came from https://github.com/mumuki/mumuki-gobstones-runner/issues/168#issuecomment-494849622 and I think it's a good idea to show that the runner is always generating that code and it's not the student's input.

By the other hand, maybe gobstones-cli shouldn't care about what the runner does and we can leave it as 'code' 

Mixed thoughts on this one. Tell me if you want me to rename it or why do you think  that "code" fits better :)

Reply:
I think it is ok as is. It's difficult to generalize a tool when you have just one use case :stuck_out_tongue: